docs/source/script_tutorial.rst

```diff
@@ -281,12 +298,14 @@ Graphical Tools
         # start Control
         control = artview.components.ComponentsControl(parent=menu)
 
-    So we know how to close windows, what about opening new ones. This is more
-    complicated, as for now some components can just be started a priory in
-    the script, but some other like :py:class:`~artview.components.RadarDisplay`
-    and :py:class:`~artview.components.ComponentsControl` have the `GUIstart`
-    method and can be started by the user at execution time, for that use the
-    Menu method :py:func:`~artview.components.Menu.addComponentMenuItem`, for
+    Yay, we know how to close windows! What about opening new ones?
+
+    This is a bit more complicated. Some components can just be started as 
+    a priori in the script. But some components like 
+    :py:class:`~artview.components.RadarDisplay` and 
+    :py:class:`~artview.components.ComponentsControl` have the `GUIstart`
+    method and can be started by the user at execution time. To do this, use the
+    Menu method :py:func:`~artview.components.Menu.addComponentMenuItem`. For
     instance
```

Comment:
I don't have a official opinion on that, what you think is better is fine with me.  `addPluginMenuItem` will never be user inside `Menu` that not from `addComponent`, that is because add a plugin is a script operation.